performQuery.js

```diff
@@ -0,0 +1,21 @@
+async function performQuery(client, context, query, values) {
```

Comment:
Lets put this file in a SharedCode or HelperFunctions folder to keep things organized. See this doc: [Azure Functions JavaScript developer guide](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node#folder-structure)